--- crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh	2021-04-05 13:55:31.047130000 +0000
+++ crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh.new	2021-04-05 13:57:13.841170000 +0000
@@ -10,7 +10,7 @@
 
 do_expat_get() {
     CT_GetFile "expat-${CT_EXPAT_VERSION}" .tar.gz    \
-               http://downloads.sourceforge.net/project/expat/expat/${CT_EXPAT_VERSION}
+               https://github.com/libexpat/libexpat/releases/download/R_${CT_EXPAT_VERSION//./_}
 }
 
 do_expat_extract() {
